chart/README.md

```diff
@@ -37,6 +37,9 @@ cluster using the [Helm](https://helm.sh) package manager.
 ## Features
 
 * Supported executors: ``LocalExecutor``, ``CeleryExecutor``, ``KubernetesExecutor``, ``LocalKubernetesExecutor``, ``CeleryKubernetesExecutor``
+* Supported AWS executors:
```

Comment:
This would be a good place to document we need x provider version, likely `8.21.0`? (cc @o-nikolas)

Reply:
If we keep the long paths, I guess the provider version can be `8.20.0`?

Reply:
Adding this as a review comment to remove my requested change. The PR with the shorter module paths is merged. Feel free to use them or not. I'll leave it up to you folks! :smiley: